src/development-status.html

```diff
@@ -4,43 +4,10 @@
 {% block content %}
  <h1 class="narrow">Status of Development</h1>
 
- <h2 class="narrow">Release Tour for the current development series</h2>
-
-<div class="txt narrow">
-
- The <a href="https://wiki.sagemath.org/ReleaseTours">release notes for the current development series</a> are developed by collaborative editing on the Sage Wiki.
+ See <a href="https://trac.sagemath.org/wiki/WikiStart#Followingthepulseofdevelopment">.
```

Comment:
this just ends up as a "dot". I'll fix this by closing the anchor with some text.

![Screenshot from 2022-05-01 19-23-11](https://user-images.githubusercontent.com/207405/166157066-77892044-b361-4b31-b700-dea71f1926d4.png)

Reply:
Thanks!